app/styles/partials/utils.scss

```diff
@@ -15,7 +15,7 @@
 }
 
 .link {
```

Comment:
If you're making it `default` you might as well remove the entire class definition here :)

Reply:
Ok alright. On it.

Reply:
@niranjan94 I have made the required changes.

Reply:
@therohitramesh I think the removal of class definition was expected here. You made it to default.

Reply:
@sumedh123 I did do that and pushed it to my fork.

Reply:
@therohitramesh Please check if you did it in the correct branch from which you made the pull request.

Reply:
@sumedh123 Can I close this Pull Request and open a new one? There is a slight problem. The fork has changed.

Reply:
I will be closing this Pull Request due to a mistake of mine. Anyway, I will reference this Pull Request number in a new Pull Request which I will be making. Sorry for the inconvenience caused.